src/elements/frame/auxiliar/GaussPointsAndWeights.m

```diff
@@ -0,0 +1,89 @@
+function [xIntPoints, wIntPoints] = GaussPointsAndWeights ( numGaussPoints )
```

Comment:
I suggeset adding the header . you can take the content from https://github.com/ONSAS/ONSAS.m/blob/master/currentFileHeader.txt

Reply:
Done